src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -230,6 +239,39 @@ function AddMutationModal({
     inputRef.current?.focus();
   }, []);
 
+  useEffect(() => {
+    async function fetchFlags() {
+      const fetchedFlags = await getFlagsByType?.(FlagTypeEnum.ALTERATION_CATEGORY);
+      if (fetchedFlags) {
+        setFlags(fetchedFlags);
+      }
+    }
+
+    fetchFlags();
+  }, []);
+
+  useEffect(() => {
+    if (flags) {
+      setSelectedStringMutationFlags(
```

Comment:
If firebase flags cannot be found in the backend, then they are not displayed. I don't imagine that this will ever happen unless we manually insert new flags through Firebase. 

I can trigger a createEntity api call for all flags that were not found in server side.

Reply:
Per our discussion, we will follow up in this ticket: https://github.com/oncokb/oncokb-pipeline/issues/536